oauth2/fosite_store_helpers.go

```diff
@@ -52,28 +53,80 @@ var defaultRequest = fosite.Request{
 	Session:           &Session{DefaultSession: &openid.DefaultSession{Subject: "bar"}},
 }
 
-func mockRequestForeignKey(t *testing.T, id string, x managerTestSetup, createClient bool) {
+var lifespan = time.Hour
```

Comment:
I had to copy theses out since they're originally found in a _test.go file here: https://github.com/ory/hydra/blob/master/oauth2/handler_test.go#L55